recipes/cppfront/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conan import ConanFile
+from conan.tools.files import save
+
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.files import apply_conandata_patches, copy, get, patch, rmdir
+from conan.tools import build
+
+import os
+import textwrap
+
+required_conan_version = ">=1.52.0"
+
+class CppfrontConan(ConanFile):
+    name = "cppfront"
+    description = "Cppfront is a experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1)"
+    topics = ("cppfront", "cpp2")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/hsutter/cppfront"
+    license = "CC-BY-NC-ND-4.0"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {"gcc": "11",
+                "Visual Studio": "16.9",
+                "clang": "13",
+                "apple-clang": "13",
+                }
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+
```

Comment:
```suggestion

    def package_id(self):
        del self.info.settings.compiler

```
https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings